tools/performance-benchmarking/src/md/0010-content.md

```diff
@@ -0,0 +1,916 @@
+`benchcomp` allows you to:
+
+* Run two or more sets of benchmark suites under different configurations;
+* declaratively define visualizations to be generated from the resulting data;
+* run the tool with sensible defaults, or hook into every stage of the pipeline when you need more flexibility.
+
+This documentation contains three sections.
+The [user walkthrough](#user-walkthrough) takes you through the process of setting up an entirely new benchmark run and dashboard.
+The [developer reference](#developer-reference) describes `benchcomp`'s architecture and the different data formats that it uses, enabling you to author a benchmark run and dashboard of your own.
+The [visualization viewer guide](#benchcomp-visualization-viewer-guide) describes how to use dashboards that `benchcomp` generates.
+Although dashboards are highly customizable, the guide describes the common elements.
+
+
+# User Walkthrough
```

Comment:
It'd be nice to have in the User Walk-though a small example that I could follow and try the tool on my on.